crates/cheatcodes/src/env.rs

```diff
@@ -235,6 +238,33 @@ impl Cheatcode for envOr_13Call {
     }
 }
 
+impl Cheatcode for isContextCall {
+    fn apply(&self, _state: &mut Cheatcodes) -> Result {
+        let Self { context } = self;
+        match context {
+            ForgeContext::TestGroup => {
+                let is_test_group = is_forge_context(&ForgeContext::Test) ||
```

Comment:
I would prefer we call 
`let Some(current_context) = FORGE_CONTEXT.get() else { return Ok(false.abi_encode()) }`
at the top of this function and then just comparing, rather than the `is_forge_context` function.